deploy/osd-suricata/05-osd-suricata-scc-SecurityContextConstraint.yaml

```diff
@@ -21,6 +21,8 @@ seLinuxContext:
   type: RunAsAny
 supplementalGroups:
   type: RunAsAny
+users:
+- suricata-sa  
```

Comment:
You need to rename the SA manifest to `04-...` so the SA is ensured to be created before applying this.

Reply:
Updated